packages/user-group-service/src/groups/resolver.ts

```diff
@@ -14,6 +15,34 @@ export const GroupResolver = {
         .limit( limit )
         .exec();
     },
+    group ( root: any, { cn }: GraphQLArgs, ctx: any ) {
+      return Groups
+        .find({ ldapCommonName: cn })
+        .exec()
+        .then( res => res[0] );
+    },
+    groups ( root: any, { selector, limit }: GraphQLArgs, ctx: any ) {
+      return Groups
+        .find( selector )
+        .limit( limit )
+        .exec();
+    },
+    groupsByUser ( root: any, args: any, ctx: any ) {
+      return UserGroupAPIHelper.roverFetch( `/groups/find/field?criteria=*${ args.uid }*&field=uniqueMember` )
+        .then( ( res: any ) => {
+          return res?.result?.result.map((group: any) => group.cn);
```

Comment:
```suggestion
          return res?.result?.result.map((group: any) => group.cn);
```

can you please use some other variable for result.result